### PR TITLE
CSS and images added 

### PR DESCRIPTION
**Pull request by CallMeSandra**
My first contact with CSS, I'm waiting for feedback :)

### CODE REVIEW

File: lib/models/product.rb [hunk 1]
```diff
@@ -1,13 +1,14 @@
 module Shop
   class Product
-    attr_reader :name, :price , :id, :quantity
+    attr_reader :name, :price , :id, :quantity, :image
```
Comment (gotar) [inline lib/models/product.rb]: you use url here, maybe better will be to name it image_url

File: public/application.css [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+  text-align: center;
+}
+
+.table_field
+{
+  margin: 0% 30%;
+}
+
+.button_submit
+{
```
Comment (gotar) [inline public/application.css]: more { to line before, like

``` css
.table {

}
```

File: views/basket/index.erb [hunk 3]
```diff
@@ -1,15 +1,15 @@
 <h1> PRZYNAJMNEJ DZIAŁA</h1>
 
 
-<table width="100%">
+<table width="90%" class="table table-condensed">
```
Comment (gotar) [inline views/basket/index.erb]: you shoudl set width in css

File: App.rb [hunk 4]
```diff
@@ -5,11 +5,11 @@
...
-    Product.new("Coffe",20),
-    Product.new("Tea",30),
-    Product.new("Chocolate",12),
-    Product.new("Cookies",50),
-    Product.new("Fireworks",300)
+    Product.new("Pikachu",8200, "http://assets.pokemon.com/assets/cms2/img/pokedex/full/025.png"),
+    Product.new("Bulbasaur",3000, "http://assets.pokemon.com/assets/cms2/img/pokedex/full/001.png"),
+    Product.new("Squirle",3900, "http://assets.pokemon.com/assets/cms2/img/pokedex/full/007.png"),
+    Product.new("Jigglypuff",2400, "http://assets.pokemon.com/assets/cms2/img/pokedex/full/039.png"),
+    Product.new("Snorlax",5800, "http://assets.pokemon.com/assets/cms2/img/pokedex/full/143.png")
```
Comment (smt116) [inline App.rb]: What if those images will be removed in the future? or blocked?

Comment (smt116) [inline App.rb]: missing spacebar before second arguments

File: App.rb [hunk 5]
```diff
@@ -62,6 +62,7 @@ class App<Sinatra::Base
       DeleteFromBasket.new(params).call
       redirect "/basket"
     end
+
```
Comment (smt116) [inline App.rb:65]: 👍

File: views/layout.erb [hunk 6]
```diff
@@ -1,14 +1,18 @@
 <html>
-
 <head>
   <title>Shop</title>
   <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.6/css/bootstrap.min.css" integrity="sha384-1q8mTJOASx8j1Au+a5WDVnPi2lkFfwwEAa8hDDdjZlpLegxhjVME1fgjWPGmkzs7" crossorigin="anonymous">
+  <link rel="stylesheet" type="text/css" href="/application.css">
+
```
Comment (smt116) [inline views/layout.erb]: useless empty line

File: views/layout.erb [hunk 7]
```diff
@@ -1,14 +1,18 @@
...
-      </html>
+<body class="container">
+  <a href="/"><img src="https://s32.postimg.org/8oytgl7vp/Logo_Pok_mon.png" alt="logo" width="90%"></a>
+  <div class="center">
+    <h1 class="center">Welcome to our shop</h1>
+    <a class="button_home" href="/">Home</a>
+    <a class="button_home" href="/basket">Basket</a>
+  </div>
+  <%= yield %>
+</body>
```
Comment (smt116) [inline views/layout.erb:15]: imho it it should be indented (all html selector childs)

File: views/products/index.erb [hunk 8]
```diff
@@ -1,8 +1,10 @@
...
-    <% end %>
-  </li>
-</ul>
+<h3 class="center"> Our products</h3>
+<div class="table_field">
+  <ul class="list-group" >
+    <% products.each do |product| %>
+    <li class="list-group-item">
+      <a href="/product/<%= product.id %>"><h4><%= product.name %></h4></a>
+      <% end %>
```
Comment (smt116) [inline views/products/index.erb:7]: indentation........

File: views/products/show.erb [hunk 9]
```diff
@@ -1,10 +1,21 @@
...
+     <h3><%= product.name %></h3>
+     <h4><b>Name:</b> <%= product.name %> </h4>
+     <h4><b>Price:</b> <%= product.price %> </h4>
+     <form action="/basket" method="post">
+      <input type="hidden" name="product_id" value=<%= product.id %>>
+      <input type="number" name="quantity" value=1>
+      <button type="submit" class="button_submit">BUY</button>
+    </form>
+  </td>
+</tr>
```
Comment (smt116) [inline views/products/show.erb]: indentation..

**Comment by smt116** [comment]
Try to fix those commit messages as `added some new` doesn't say anything. Don't use past tenses in commit title.

File: App.rb (hunk 4 above)
Comment (CallMeSandra) [inline App.rb]: @smt116 I couldn't add pictures from my project folder, so I choose url. But I see that using an url from other sites is a bad practise; I'll use image hosting next time.

Comment (smt116) [inline App.rb]: you can put them into your repository to `public/images` directory.

**Pull request merged**